doc/nrf/ug_matter_overview_device_types.rst

```diff
@@ -0,0 +1,210 @@
+| Unit              | or off by means of a bound controller device such as an on/off light switch  |                     |                                       |
+|                   | or a non-color controller. The On/Off Plug-in Unit is then capable of having |                     |                                       |
+|                   | a non-communicating light attached to it.                                    |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Dimmable Plug-In  | A Dimmable Plug-In Unit is a device that is capable of being switched on     | Certifiable         |                                       |
+| Unit              | or off and have its level adjusted by means of a bound controller device     |                     |                                       |
+|                   | such as a dimmer switch or a non-color controller. The device is then        |                     |                                       |
+|                   | capable of having a non-communicating light attached to it.                  |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Pump              | A Pump device is a pump that may have variable speed. It may have optional   | Certifiable         |                                       |
+|                   | built-in sensors and a regulation mechanism. It is typically used            |                     |                                       |
+|                   | for pumping fluids like water.                                               |                     |                                       |
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+
+.. _ug_matter_device_types_switches_controls:
+
+Switches and Controls device types
+==================================
+
++-------------------+------------------------------------------------------------------------------+---------------------+---------------------------------------+
+| Device type       | Description (from Device Library specification)                              | State of the device | Dedicated sample in the |NCS|         |
++===================+==============================================================================+=====================+=======================================+
+| On/Off Light      | An On/Off Light Switch is a controller device that,                          | Certifiable         | :ref:`matter_light_switch_sample`     |
```

Comment:
The same story as for the Light bulb.
The Light switch supports all necessary clusters to be validated as Dimmer Switch.

Reply:
Removed as example for On/Off Light Switch.